### PR TITLE
Added Cloud Computing projects related to issue-923

### LINKED ISSUE #923: [Add] <Cloud Computing Projects>
### Site URL

https://www.interviewbit.com/blog/cloud-computing-projects/

### Description of the above provided site

This website provides free Cloud Computing projects for beginners to advanced with source code.

### Category

Resources

### Sub Category

Project Ideas

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by SargamPuram**
PR fixes an open issue , Closes #923 
It adds Cloud Computing projects in project-ideas of resources .
Kindly review this
Screenshot
LH3

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
LGTM!

File: database/resources/project-ideas.json [hunk 1]
```diff
@@ -46,5 +46,12 @@
     "url": "https://www.interviewbit.com/blog/artificial-intelligence-projects/",
     "category": "resources",
     "subcategory": "project-ideas"
+  },
+  {
+    "name": "Cloud Computing Projects",
+    "description": "This website provides free Cloud Computing projects for beginners to advanced with source code.",
```
Comment (rupali-codes) [inline database/resources/project-ideas.json:52]:
```suggestion
    "description": "This website provides free Cloud Computing projects for beginners to advanced level with source code.",
```

Comment (SargamPuram) [reply database/resources/project-ideas.json]: @rupali-codes yes , I've made the necessary changes.
Kindly review it.

Comment (rupali-codes) [reply database/resources/project-ideas.json]: thanks

**Pull request merged**